feature-detects/css/animations.js

```diff
@@ -8,7 +8,7 @@
   "warnings": ["Android < 4 will pass this test, but can only animate a single property at a time"],
   "notes": [{
     "name" : "Article: 'Dispelling the Android CSS animation myths'",
-    "href": "http://goo.gl/OGw5Gm"
+    "href": "https://daneden.me/2011/12/14/putting-up-with-androids-bullshit/"
```

Comment:
Fixed.

> We actually short url'd this because the use of bullshit didn't pass some check in visual studio believe it or not https://plus.google.com/u/0/+shanselman/posts/Kfq6eWrueCz?cfem=1

@ralphholzmann Interesting. BTW: that URL doesn't work for me. :(

![](https://cloud.githubusercontent.com/assets/1223565/12342082/97f067a2-bb2f-11e5-9ba0-c164cc1c682d.png)